libp2p/transports/tcptransport.nim

```diff
@@ -77,12 +77,9 @@ proc connHandler*(t: TcpTransport,
       trace "Cleaning up client", addrs = $client.remoteAddress,
                                   conn
 
-      if not(isNil(conn) and conn.closed()):
-        await conn.close()
-
       t.clients[dir].keepItIf( it != client )
-      if not(isNil(client) and client.closed()):
-        await client.closeWait()
+      await allFuturesThrowing(
```

Comment:
double all?

Reply:
heh, I keep tripping on this one not sure why.

Reply:
too many magic helpers

Reply:
true